library/ssl_tls13_client.c

```diff
@@ -132,9 +132,9 @@ static int ssl_tls13_write_client_hello_cipher_suites(
             unsigned char *end,
             size_t *olen )
 {
+    unsigned char *p = buf;  /* Iteration over the cipher_suites list */
```

Comment:
```suggestion
    unsigned char *p = buf;
```
This is just the writing pointer as we aim to do it in TLS code thus no need for extra comment.